registration/tests/test_api.py

```diff
@@ -430,6 +430,53 @@ def test_theme_config_api_erase_value(self):
         )
         self.assertNotIn("btn-sign-in-bg", self.instance_config.draft_theme_config.keys())
 
+    def test_static_templates_overrides_config_when_no_existing_config(self):
+        """
+        Test updating the static templates overrides configuration when there is no existing configuration.
+        """
+        self.client.force_login(self.user_with_instance)
+        self._setup_user_instance()
+
+        response = self.client.patch(
+            reverse(
+                f"api:v2:openedx-instance-config-static-templates-overrides-config",
+                args=(self.instance_config.pk, )
+            ),
+            data={'version': 1},
```

Comment:
Isn't this supposed to be version 0? Or are you specifically trying to make the API error out here?

Reply:
Btw, if there's no template overrides set, and I make a request, will it error out?

Reply:
> Isn't this supposed to be version 0? Or are you specifically trying to make the API error out here?

Yes, you are right. This looks like a typo. I will fix it :+1: 

> Btw, if there's no template overrides set, and I make a request, will it error out?

Yes, with the current implementation, it will error out. However, the [RFC 5789](https://tools.ietf.org/html/rfc5789) for the `PATCH` method states 

```
If the Request-URI does not point to an existing resource, the server MAY create a new resource,
depending on the patch document type (whether it can logically modify a null resource)
and permissions, etc.
```
In this case, we could either allow the `PATCH` method to treat this as a missing resource and create it if it doesn't exist or treat the default `null` value as the resource being present and allow patching. Either way the implementation will be the same.

Reply:
@giovannicimolin, I have updated the `PATCH` method to automatically update the `None` default value to the data passed in the `PATCH` request and also add the version if it is not present in the request data.